tests/test_cli_utils.py

```
@@ -391,3 +393,442 @@ def test_handle_matrix_error_unknown(self, mock_logger):
+        with (
+            patch(
+                "mmrelay.matrix_utils.load_credentials", return_value=mock_credentials
+            ),
+            patch("mmrelay.cli_utils.AsyncClient") as mock_async_client,
+            patch("mmrelay.cli_utils._create_ssl_context", return_value=None),
+            patch("mmrelay.config.save_credentials") as mock_save_creds,
+            patch("mmrelay.cli_utils._cleanup_local_session_data", return_value=True),
+        ):
+            mock_async_client.side_effect = [mock_temp_client, MagicMock()]
+
+            result = await logout_matrix_bot(password="test_password")
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)